tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
 "org.eclipse.jdt:org.eclipse.jdt.core:3.10.0",z,EPL-1.0
 "org.eclipse.osgi:org.eclipse.osgi:3.7.1",z,EPL-1.0
 "org.eclipse.text:org.eclipse.text:3.5.101",z,EPL-1.0
+"elasticsearch-api:5.0.4",https://github.com/elastic/elasticsearch-ruby,Apache-2.0
+"elasticsearch-transport:5.0.4",https://github.com/elastic/elasticsearch-ruby,Apache-2.0
+"elasticsearch:5.0.4",https://github.com/elastic/elasticsearch-ruby,Apache-2.0
+"equalizer:0.0.10",https://github.com/dkubb/equalizer,MIT
+"faraday:0.9.2",https://github.com/lostisland/faraday,MIT
+"ffi:1.9.23",https://github.com/ffi/ffi,BSD-3-CLAUSE
+"filesize:0.0.4",https://github.com/dominikh,MIT
+"filewatch,:0.9.0",https://github.com/jordansissel/ruby-filewatch,MIT
+"ftw:0.0.48",https://github.com/jordansissel/ruby-ftw,Apache-2.0
+"gelf:3.0.0",https://github.com/graylog-labs/gelf-rb,MIT
+"gelfd:0.2.0",https://github.com/lusis/gelfd,Apache-2.0
+"gems:0.8.3",https://github.com/rubygems/gems,MIT
+"gmetric:0.1.3",https://github.com/igrigorik/gmetric,MIT
+"gserver:0.0.1",https://github.com/ruby/gserver/blob,Ruby
+"hitimes:1.2.6",https://github.com/copiousfreetime/hitimes,ISC
+"http-cookie:1.0.3",https://github.com/sparklemotion/http-cookie,MIT
+"http-form_data:1.0.1",https://github.com/httprb/form_data,MIT
+"http:0.9.9",https://github.com/httprb/http,MIT
+"http_parser.rb:0.6.0",https://github.com/tmm1/http_parser.rb,MIT
+"i18n:0.6.9",https://github.com/svenfuchs/i18n,MIT
+"insist:1.0.0",https://github.com/jordansissel/ruby-insist,Apache-2.0
+"com.headius:invokebinder:1.7",https://github.com/headius/invokebinder,Apache-2.0
+"jar-dependencies:0.3.12",https://github.com/mkristian/jar-dependencies,MIT
+"org.jruby:jay-yydebug:-",https://github.com/jruby/jay-yydebug,EPL-2.0|GPL-2.0|LGPL-2.1
```

Comment:
This library is referenced in [jruby licensing text](https://github.com/jruby/jruby/blob/9.1.13.0/COPYING), but I could not find the version used anywhere, not even inside the `jruby-complete.jar`